sample-kotlin/src/main/java/com/infullmobile/android/infullmvp/sample/activity/SampleActivityView.kt

```diff
@@ -6,12 +6,15 @@ import android.widget.TextView
 
 import com.infullmobile.android.infullmvp.PresentedActivityView
 import com.infullmobile.android.infullmvp.sample.R
+import com.infullmobile.android.infullmvp.sample.sample_mvp_card.SampleMvpCard
 
-open class SampleActivityView(private val pagerAdapter: FragmentStatePagerAdapter)
-    : PresentedActivityView<SampleActivityPresenter>() {
+open class SampleActivityView(
+        private val pagerAdapter: FragmentStatePagerAdapter
+) : PresentedActivityView<SampleActivityPresenter>() {
 
     val textField: TextView by bindView(R.id.textContainer)
     val pager: ViewPager by bindView(R.id.pagesContainer)
+    val customView: SampleMvpCard by bindView(R.id.customView)
```

Comment:
Why not just create other sample for the custom view specifically?

If it is a simple boiler plate class, lets keep it that way.